users/factories.py

```diff
@@ -2,6 +2,14 @@
 import factory.fuzzy
 
 from users.models import CustomUser
+from users.models import CustomUserPreferences
+
+
+class UserPreferencesFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = CustomUserPreferences
+
+    user = factory.SubFactory("users.factories.UserFactory", preferences=None)
```

Comment:
Why you adding here `preferences=None`?

Reply:
`SubFactory` is used in `UserPreferencesFactory` with `preferences=None` to prevent creating new preferences when calling `UserPreferencesFactory()` standalone. 
I based this implementation on [documentation](https://factoryboy.readthedocs.io/en/stable/recipes.html)